### PR TITLE
Feature/home

### PR DESCRIPTION
**Pull request by gdethier**

### CODE REVIEW

File: src/common/Wallet.tsx [hunk 1]
```diff
@@ -52,66 +52,64 @@ export function Content(props: Props) {
...
+                            }
+                        ]}
+                        data={ balances }
+                        renderEmpty={ () => <EmptyTableMessage>You have no asset yet</EmptyTableMessage> }
+                    />
+                </Frame>
+            </Col>
+            <Col md={4}>
+                <Frame
+                    title="Current LGNT balance"
```
Comment (benoitdevos) [inline src/common/Wallet.tsx:101]: We should use `Balances.SYMBOL` here as well.

File: src/common/WalletGauge.tsx [hunk 2]
```diff
@@ -153,7 +153,7 @@ export default function WalletGauge(props: Props) {
                 ]}
                 size="lg"
             >
-                <h3>Transfer LOGs</h3>
+                <h3>Transfer LGNTs</h3>
```
Comment (benoitdevos) [inline src/common/WalletGauge.tsx:156]: Same remark.

File: src/logion-chain/Balances.ts [hunk 3]
```diff
@@ -100,7 +101,7 @@ export function getCoin(coinId: string): Coin {
             name: 'Logion',
             iconId: 'log',
             iconType: 'png',
-            symbol: 'LOG',
+            symbol: 'LGNT',
```
Comment (benoitdevos) [inline src/logion-chain/Balances.ts:104]: Same remark.

**Review by benoitdevos — APPROVED** [review]
Good job ! A minor suggestion:

File: src/common/Wallet.tsx (hunk 1 above)
Comment (gdethier) [reply src/common/Wallet.tsx:101]: Indeed!

**Pull request merged**